### ISSUE TITLE
1.0 tag is broken for django 1.9.X

### ISSUE THREAD

**Issue by dudanogueira** [caktus/django-treenav#66]
Hi! I was trying to figure out what's happening but couldn't. the traceback is:

web_1 | Traceback (most recent call last):
web_1 |   File "manage.py", line 10, in <module>
web_1 |     execute_from_command_line(sys.argv)
web_1 |   File "/usr/local/lib/python2.7/dist-packages/django/core/management/**init**.py", line 350, in execute_from_command_line
web_1 |     utility.execute()
web_1 |   File "/usr/local/lib/python2.7/dist-packages/django/core/management/**init**.py", line 342, in execute
web_1 |     self.fetch_command(subcommand).run_from_argv(self.argv)
web_1 |   File "/usr/local/lib/python2.7/dist-packages/django/core/management/**init**.py", line 176, in fetch_command
web_1 |     commands = get_commands()
web_1 |   File "/usr/local/lib/python2.7/dist-packages/django/utils/lru_cache.py", line 100, in wrapper
web_1 |     result = user_function(_args, *_kwds)
web_1 |   File "/usr/local/lib/python2.7/dist-packages/django/core/management/**init**.py", line 71, in get_commands
web_1 |     for app_config in reversed(list(apps.get_app_configs())):
web_1 |   File "/usr/local/lib/python2.7/dist-packages/django/apps/registry.py", line 137, in get_app_configs
web_1 |     self.check_apps_ready()
web_1 |   File "/usr/local/lib/python2.7/dist-packages/django/apps/registry.py", line 124, in check_apps_ready
web_1 |     raise AppRegistryNotReady("Apps aren't loaded yet.")
web_1 | django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.

I was able to reproduce the same error on the example project, just by upgrading the django version to 1.9.1

Thanks!

**Comment by benred42** [comment]
This might be caused by the placement of ```default_app_config``` in treenav's ```init.py```.  According to django's documentation (https://docs.djangoproject.com/en/1.9/ref/applications/#for-application-authors) you can end up with a namespace conflict.

**Comment by davemutton** [comment]
I am having the same issue on django 1.9.7 and django-treenav 1.0.0

**Comment by davemutton** [comment]
@dudanogueira  This might be a little hacky as a patch but in order to get it working I installed the treenav app locally and then made the following edits to signals.py

Line 1: Delete "#from django.contrib.contenttypes.models import ContentType"
Line 25: Replace with "        ct = django.contrib.contenttypes.models.ContentType.objects.get_for_model(sender)
"


hope this helps till a patch is issued.

**Comment by dborisog** [comment]
It seems a similar issue.

```
Django==1.10.2
django-mptt==0.8.6
django-treenav==1.0.0
```

```
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7f104faf92f0>
Traceback (most recent call last):
  File ".../django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File ".../django/core/management/commands/runserver.py", line 113, in inner_run
    autoreload.raise_last_exception()
  File ".../django/utils/autoreload.py", line 249, in raise_last_exception
    six.reraise(*_exception)
  File ".../django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File ".../django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File ".../django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File ".../django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File ".../django/apps/config.py", line 116, in create
    mod = import_module(mod_path)
  File ".../python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File ".../treenav/apps.py", line 4, in <module>
    from .signals import connect_post_save_handler, disconnect_post_save_handler
  File ".../treenav/signals.py", line 1, in <module>
    from django.contrib.contenttypes.models import ContentType
  File ".../django/contrib/contenttypes/models.py", line 138, in <module>
    class ContentType(models.Model):
  File ".../django/db/models/base.py", line 105, in __new__
    app_config = apps.get_containing_app_config(module)
  File ".../django/apps/registry.py", line 237, in get_containing_app_config
    self.check_apps_ready()
  File ".../django/apps/registry.py", line 124, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.

```

**Comment by vkurup** [comment]
Thanks for the reports. This should be fixed in the new 1.1.0 release: https://pypi.python.org/pypi/django-treenav/1.1.0
